services/blockchain-connector/shared/sdk/nft.js

```diff
@@ -37,6 +41,46 @@ const getNFTConstants = async () => {
 	}
 };
 
+// TODO: Remove lint suppression once sdk endpoint is available
+// eslint-disable-next-line no-unused-vars
+const updateCollectionIDs = async (params) => {
+	try {
+		// TODO: Invoke sdk endpoint once available
+		// moduleConstants = await invokeEndpoint('nft_getCollectionIDs',{ chainID: params.chainID });
+		supportedCollectionIDsInfo = { collectionIDs: ['*'] };
+
+		return supportedCollectionIDsInfo;
+	} catch (err) {
+		if (err.message.includes(timeoutMessage)) {
+			throw new TimeoutException('Request timed out when calling \'updateCollectionIDs\'.');
+		}
+		logger.warn(`Error returned when invoking 'nft_getCollectionIDs'.\n${err.stack}`);
+		throw err;
+	}
+};
+
+const getCollectionIDs = async () => supportedCollectionIDsInfo;
+
+const updateCollectionIDsOnBlockChange = async () => {
```

Comment:
Please re-verify if this is really necessary.

Reply:
Applied.